sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/api/Azure.ResourceManager.NeonPostgres.net8.0.cs

```diff
@@ -190,6 +658,24 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public static bool operator !=(Azure.ResourceManager.NeonPostgres.Models.NeonResourceProvisioningState left, Azure.ResourceManager.NeonPostgres.Models.NeonResourceProvisioningState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class NeonRoleProperties : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NeonPostgres.Models.NeonRoleProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.NeonRoleProperties>
+    {
+        public NeonRoleProperties() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.NeonPostgres.Models.Attributes> Attributes { get { throw null; } }
+        public string BranchId { get { throw null; } set { } }
+        public string CreatedAt { get { throw null; } }
```

Comment:
would this be a utc time or a date time?
if so, I think we should change it in our spec to let it be a `date-time` of your format - but this could be a breaking change to your spec and to other languages' SDK.